modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -1423,6 +1423,102 @@ describe('V2 Wallet:', function () {
     });
   });
 
+  describe('manage unspents', function () {
+    let rootWalletKey;
+    let walletPassphrase;
+    let basecoin;
+    let wallet;
+    let keysObj;
+
+    before(async function () {
+      rootWalletKey = getDefaultWalletKeys();
+      walletPassphrase = 'fixthemoneyfixtheworld';
+      keysObj = toKeychainObjects(rootWalletKey, walletPassphrase);
+      basecoin = bitgo.coin('tbtc');
+      const walletData = {
+        id: '5b34252f1bf349930e34020a',
+        coin: 'tbtc',
+        keys: keysObj.map((k) => k.id),
+      };
+      wallet = new Wallet(bitgo, basecoin, walletData);
+    });
+
+    it('should pass for bulk consolidating unspents', async function () {
+      const psbts = [0, 2].map((_) =>
+        utxoLib.testutil.constructPsbt(
+          [{ scriptType: 'p2wsh', value: BigInt(1000) }],
+          [{ scriptType: 'p2shP2wsh', value: BigInt(900) }],
+          basecoin.network,
+          rootWalletKey,
+          'unsigned'
+        )
+      );
+      const txHexes = psbts.map((psbt) => ({ txHex: psbt.toHex() }));
+
+      const nocks: nock.Scope[] = [];
+      nocks.push(
+        nock(bgUrl).post(`/api/v2/${wallet.coin()}/wallet/${wallet.id()}/consolidateUnspents`).reply(200, txHexes)
+      );
+
+      nocks.concat(
+        keysObj.map((k, i) => nock(bgUrl).get(`/api/v2/${wallet.coin()}/key/${wallet.keyIds()[i]}`).reply(200, k))
+      );
```

Comment:
concat has no side effect
```suggestion
      nocks.push(
        ...keysObj.map((k, i) => nock(bgUrl).get(`/api/v2/${wallet.coin()}/key/${wallet.keyIds()[i]}`).reply(200, k))
      );
```